src/listen/websocket.rs

```diff
@@ -69,18 +68,18 @@ impl Transcription<'_> {
     where
         S: Stream<Item = std::result::Result<Bytes, E>>,
     {
-        self.stream_request_with_options(None)
+        self.stream_request_with_options(Options::builder().build())
     }
 
-    pub fn stream_request_with_options<'a, E, S>(
-        &'a self,
-        options: Option<&'a Options>,
-    ) -> StreamRequestBuilder<'a, S, E>
+    pub fn stream_request_with_options<E, S>(
+        &self,
+        options: Options,
+    ) -> StreamRequestBuilder<'_, S, E>
     where
         S: Stream<Item = std::result::Result<Bytes, E>>,
     {
         StreamRequestBuilder {
-            config: self.0,
+            deepgram: self.0,
```

Comment:
I changed this because I was getting confused about what a config is vs an options.